include/cantera/kinetics/StoichManager.h

```diff
@@ -177,7 +177,7 @@ class C1
     //! Species number
     size_t m_ic0;
 
-    size_t m_jc0; //!< Index in derivative triplet vector
+    int m_jc0; //!< Index in derivative triplet vector
```

Comment:
This is used as an index into a `std::vector`, so I think `size_t` is still appropriate, right?

Reply:
True. Was overthinking this, changed all of these back.